ReactiveCocoa/UIKit/iOS/UIPickerView.swift

```diff
@@ -0,0 +1,43 @@
+import ReactiveSwift
+import enum Result.NoError
+import UIKit
+
+private class PickerViewDelegateProxy: DelegateProxy<UIPickerViewDelegate>, UIPickerViewDelegate {
+	@objc func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+		forwardee?.pickerView?(pickerView, didSelectRow: row, inComponent: component)
+	}
+}
+
+extension Reactive where Base: UIPickerView {
+
+	private var proxy: PickerViewDelegateProxy {
+		return .proxy(for: base,
+		              setter: #selector(setter: base.delegate),
+		              getter: #selector(getter: base.delegate))
+	}
+
+	/// Sets the selected row in the specified component, without animating the
+	/// position.
+	public func selectedRow(inComponent component: Int) -> BindingTarget<Int> {
+		return makeBindingTarget { $0.selectRow($1, inComponent: component, animated: false) }
+	}
+
+	/// Reloads all components
+	public var reloadAllComponents: BindingTarget<()> {
+		return makeBindingTarget { base, _ in base.reloadAllComponents() }
+	}
+
+	/// Reloads the specified component
+	public var reloadComponent: BindingTarget<Int> {
+		return makeBindingTarget { $0.reloadComponent($1) }
+	}
+
+	/// Create a signal which sends a `value` event for each row selection
+	///
+	/// - returns:
+	///   A trigger signal.
+	public var selections: Signal<(Int, Int), NoError> {
```

Comment:
The same had occurred to me, so I think it would – otherwise the order of row and component is ambiguous. I think a new type would be clearer than `NSIndexPath`.